staging/vhost-device-gpu/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "vhost-device-gpu"
+version = "0.1.0"
+authors = ["Dorinda Bassey <dbassey@redhat.com>", "Matej Hrica <mhrica@redhat.com>"]
+description = "A virtio-gpu device using the vhost-user protocol."
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["gpu", "vhost", "vhost-user", "virtio"]
+categories = ["multimedia::video", "virtualization"]
+license = "Apache-2.0 OR BSD-3-Clause"
+edition = "2021"
+publish = false
+
+[features]
+xen = ["vm-memory/xen", "vhost/xen", "vhost-user-backend/xen"]
+
+[dependencies]
+clap = { version = "4.4", features = ["derive"] }
+env_logger = "0.11.5"
+libc = "0.2"
+log = "0.4"
+
+[target.'cfg(not(target_env = "musl"))'.dependencies]
+rutabaga_gfx = { version = "0.1.4", features = ["gfxstream", "virgl_renderer"] }
+thiserror = "1.0"
+vhost = { version = "0.12.1", features = ["vhost-user-backend"] }
+vhost-user-backend = "0.16.1"
+virtio-bindings = "0.2.2"
+virtio-queue = "0.13.0"
+vm-memory = "0.15.0"
+vmm-sys-util = "0.12.1"
```

Comment:
Seems like the rebase happened upon commits that updated various rust-vmm deps (`vhost`, `vm-memory` etc for other crates. Could you update them here as well? It's not an issue, but it will save you updating them on a future PR or dependabot complaining about it with a PR of its own

Reply:
@dorindabassey please check also the CI, some `staging` pipelines are failing.
The one of no-staging I think it's related to pipewire (will be nice if you can check also that in the future https://github.com/rust-vmm/vhost-device/issues/647), I'm restarting that one to have at least grean CI.

Reply:
Yes please, align at least `rust-vmm` dependencies to the other crates.